frame/pablo/src/lib.rs

```diff
@@ -592,7 +678,53 @@ pub mod pallet {
 
 					Ok(base_amount_excluding_fees)
 				},
+				ConstantProduct(constant_product_pool_info) => {
+					let (base_amount, quote_amount, lp_fees, owner_fees) = Self::do_compute_swap(
+						pool_id,
+						pair,
+						quote_amount,
+						true,
+						constant_product_pool_info.fee,
+						constant_product_pool_info.owner_fee,
+					)?;
+					let total_fees = lp_fees.safe_add(&owner_fees)?;
```

Comment:
this is contradicting to do_compute_swap(), as noted if we want we can specialize fee computation method.

Reply:
Yes, got a bit confused by `do_compute_swap` function in this file.

Reply:
fixed

Reply:
so basically we want to specialize dp_compute_swap(). Okay then in next PR I will create it in stable_swap.